barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/protogalaxy_recursive_verifier.cpp

```diff
@@ -0,0 +1,305 @@
+#include "protogalaxy_recursive_verifier.hpp"
+#include "barretenberg/polynomials/polynomial.hpp"
+#include "barretenberg/proof_system/library/grand_product_delta.hpp"
+namespace bb::stdlib::recursion::honk {
+
+template <class VerifierInstances>
+void ProtoGalaxyRecursiveVerifier_<VerifierInstances>::receive_accumulator(const std::shared_ptr<Instance>& inst,
```

Comment:
I appreciate that the briefs for these functions are in the header but many of the methods themselves don't have a single comment :). A lot of the logic in them is boiler-plate-ish and pretty easy to follow when you read it but, for example, a "`// Extract relation parameters from transcript`" is a simple addition that makes reading even easier. Some of the logic is less trivial and definitely deserves a comment or two.

Reply:
Yup, makes total sense, quite glad to have a new pair of eyes on the code, these functions have evolved in size in time as we got more and more folding merged, will add some more comments